docs/faq.md

```diff
@@ -12,6 +12,28 @@ hr {
 }
 </style>
 
+<hr id="operation-buffering-timed-out" />
+
+<a class="anchor" href="#operation-buffering-timed-out">**Q**</a>. Operation `...` timed out after 10000 ms. What gives?
+
+**A**. At its core, this issue stems from being unable to connect to mongoose. If you are using an application, like a bot,
+that wishes to connect to mongoose, you must ensure that you are connecting before the bot bot activates. For example, when working
+with discord bots, the last line of your index file is `client.login(token)`. You want to place the connection string, `mongoose.connect().then()` before this line.
```

Comment:
Neither of these is true, you can start using Mongoose before connecting. You just need to make sure you actually connect at some point.

The most common cause of this issue is cases where you don't call `connect()` on the connection your models are defined on. For example:

```javascript
await mongoose.createConnection(mongodbUri);

const Test = mongoose.model('Test', schema);

await Test.findOne(); // Will throw "Operation timed out" error because didn't call `mongoose.connect()`
```

Or:

```javascript
await mongoose.connect(mongodbUri);

const db = mongoose.createConnection();

const Test = db.model('Test', schema);

await Test.findOne(); // Will throw "Operation timed out" error because `db` isn't connected to MongoDB
```

Add link to [multiple connections docs](https://mongoosejs.com/docs/connections.html#multiple_connections)